{
  "uid" : "92f0ac7a567add2c",
  "name" : "test_season_source_score_gear_comparison_false",
  "fullName" : "test_case.test_season_monitoring.test_season.TestSeason#test_season_source_score_gear_comparison_false",
  "historyId" : "b020b02b1f872c02ce5f0615492c709f",
  "time" : {
    "start" : 1683269449756,
    "stop" : 1683269607499,
    "duration" : 157743
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: [error_code] Expected <200000> to be equal to <0>, but was not.",
  "statusTrace" : "self = <test_case.test_season_monitoring.test_season.TestSeason object at 0x104ccd760>\n\n    def test_season_source_score_gear_comparison_false(self):\n        season_round_dict = {}\n        for i in self.test_data.get(\"season_round\"):\n            season_round_dict[i.get(\"label\")] = i.get(\"value\")\n        respon = HttpEastBlueApi().web_api_data_report_season_funnel(season_round_dict,[\"2023-01-02 00:00:00\", \"2023-04-05 23:59:59\"],is_level_or_gear=False)\n>       assert_that(respon.get(\"error_code\"), \"error_code\").is_equal_to(\"0\")\nE       AssertionError: [error_code] Expected <200000> to be equal to <0>, but was not.\n\ntest_case/test_season_monitoring/test_season.py:143: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: [error_code] Expected <200000> to be equal to <0>, but was not.",
    "statusTrace" : "self = <test_case.test_season_monitoring.test_season.TestSeason object at 0x104ccd760>\n\n    def test_season_source_score_gear_comparison_false(self):\n        season_round_dict = {}\n        for i in self.test_data.get(\"season_round\"):\n            season_round_dict[i.get(\"label\")] = i.get(\"value\")\n        respon = HttpEastBlueApi().web_api_data_report_season_funnel(season_round_dict,[\"2023-01-02 00:00:00\", \"2023-04-05 23:59:59\"],is_level_or_gear=False)\n>       assert_that(respon.get(\"error_code\"), \"error_code\").is_equal_to(\"0\")\nE       AssertionError: [error_code] Expected <200000> to be equal to <0>, but was not.\n\ntest_case/test_season_monitoring/test_season.py:143: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "d664397fbfded3e9",
      "name" : "login",
      "source" : "d664397fbfded3e9.txt",
      "type" : "text/plain",
      "size" : 1877
    }, {
      "uid" : "281a22a0c070dbd4",
      "name" : "[api-path]/web-api/data-report/season-funnel ",
      "source" : "281a22a0c070dbd4.txt",
      "type" : "text/plain",
      "size" : 9307
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "test_case.test_season_monitoring"
  }, {
    "name" : "suite",
    "value" : "test_season"
  }, {
    "name" : "subSuite",
    "value" : "TestSeason"
  }, {
    "name" : "host",
    "value" : "enhengdeMacBook-Air.local"
  }, {
    "name" : "thread",
    "value" : "4054-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_case.test_season_monitoring.test_season"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "92f0ac7a567add2c.json",
  "parameterValues" : [ ]
}